docs/tutorial/part-seven/index.md

```diff
@@ -1,61 +1,52 @@
 ---
-title: Programmatically create pages from data
+शीर्षक: प्रोग्रामेटिक रूप से डेटा से पेज बनाना
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/), [part 5](/tutorial/part-five/), and [part 6](/tutorial/part-six/) before continuing here.
+> यह ट्यूटोरियल Gatsby के डेटा लेयर के बारे में एक श्रंखला का हिस्सा है। आगे बढ़ने से पहले सुनिश्चित करें कि आप इससे पहले [भाग 4](/tutorial/part-four/), [भाग 5](/tutorial/part-five/), और [भाग 6](/tutorial/part-six/) पढ़ चुके है।
 
-## What's in this tutorial?
+## इस ट्यूटोरियल में क्या है?
 
-In the previous tutorial, you created a nice index page that queries markdown
-files and produces a list of blog post titles and excerpts. But you don't want to just see excerpts, you want actual pages for your
-markdown files.
+पिछले ट्यूटोरियल में, आपने एक अच्छा इंडेक्स पेज बनाया था, जो मार्कडाउन फाइल्स को क्वेरी करता है
+और ब्लॉग पोस्ट शीर्षक और अंशों की एक सूची तैयार करता है। लेकिन आप केवल अंश नहीं देखना चाहते हैं,आप मार्कडाउन फाइलें के लिए वास्तविक पेजेज चाहते हैं।
 
-You could continue to create pages by placing React components in `src/pages`. However, you'll
-now learn how to _programmatically_ create pages from _data_. Gatsby is _not_
-limited to making pages from files like many static site generators. Gatsby lets
-you use GraphQL to query your _data_ and _map_ the query results to _pages_—all at build
-time. This is a really powerful idea. You'll be exploring its implications and
-ways to use it for the remainder of this part of the tutorial.
+आप `src/pages` में React कौम्पोनॅन्टस को रखकर पेज बनाना जारी रख सकते हैं। हालाँकि, अब आप _प्रोग्रामेटिक_ रूप से _डेटा_ से पेज बनाना सीखेंगे। Gatsby स्टॅटिक साइट जनरेटर की तरह सिर्फ़ फ़ाइलों से पेजेज बनाने तक सीमित नही है। Gatsby GraphQL की मदद से _डेटा_ को क्वेरी करने देता है और क्वेरी रिज़ल्ट्स को पेजेज मे परिवर्तित कर देता है, बिल्ड टाइम मे। यह वास्तव में शक्तिशाली विचार है। आप ट्यूटोरियल के इस भाग मे इसका उपयोग करने के तरीके सीखेंगे।
```

Comment:
`क्वेरी रिज़ल्ट्स को पेजेज मे परिवर्तित कर देता है, बिल्ड टाइम मे` => `बिल्ड टाइम मे क्वेरी रिज़ल्ट्स को पेजेज मे परिवर्तित कर देता है,`

Reply:
let's get this fixed, before continuing further

Reply:
fixed